app/notification_email.go

```diff
@@ -113,7 +113,7 @@ func (a *App) sendNotificationEmail(c request.CTX, notification *PostNotificatio
 		return errors.Wrap(err, "unable to render the email notification template")
 	}
 
-	templateString := "<%s@mattermost.com>"
+	templateString := "<%s@" + a.Srv().MailServiceConfig().Hostname + ">"
```

Comment:
Instead of this, you can just call `utils.GetHostnameFromSiteURL(a.GetSiteURL())`

Reply:
Done